src/azure-cli/azure/cli/command_modules/network/_params.py

```diff
@@ -598,6 +598,7 @@ def load_arguments(self, _):
         c.argument('connection_name', options_list=['--name', '-n'], help='ExpressRoute connection name.', id_part='child_name_1')
         c.argument('routing_weight', help='Routing weight associated with the connection.', type=int)
         c.argument('authorization_key', help='Authorization key to establish the connection.')
+        c.argument('enable_internet_security', options_list='--internet-security', arg_type=get_three_state_flag(), help='Enable internet security.', min_api='2019-09-01')
```

Comment:
I suggest --enable-internet-security

Reply:
The help message, which is same as parameter name, tells nothing-_-||

Reply:
some of the arguments are like --internet-security, which help message is Enable/Disable internet security.

Reply:
already have --internet-security in other commands and service team insist on such name.

Reply:
asking for help information right now.